### DISCUSSION TITLE
ENAMETOOLONG error when starting Graphql

### DISCUSSION THREAD

**Discussion by Fainaru** [MichalLytek/typegraphql-prisma#432] [Q&A]
When starting with npm run dev my gql-yoga server gives me ENAMETOOLONG error

![Captura de pantalla 2023-12-24 a la(s) 04 12 17](https://github.com/MichalLytek/typegraphql-prisma/assets/72484366/4f083ec0-a37a-4cd2-b22d-ce8b195b0d43)
![Captura de pantalla 2023-12-24 a la(s) 04 12 33](https://github.com/MichalLytek/typegraphql-prisma/assets/72484366/ab9cd938-5d1c-4b2f-a3cb-3eb30d30de23)

**Comment by MichalLytek** [comment]
Use shorter table names

**Comment by DennisGoldfinger** [comment]
This is a `ts-node-dev` [issue](https://github.com/wclr/ts-node-dev/issues/340).
